pelita/utils.py

```diff
@@ -55,21 +52,74 @@ def load_builtin_layout(layout_name, *, is_blue=True):
     return layout_as_str(**layout_for_team(parse_layout(get_layout_by_name(layout_name)), is_blue=is_blue))
 
 
-def setup_test_game(*, layout, game=None, is_blue=True, round=None, score=None, seed=None,
+def setup_test_game(*, layout, is_blue=True, round=None, score=None, seed=None,
                     food=None, bots=None, enemy=None, is_noisy=None):
-    """Returns the first bot object given a layout.
-
-    The returned Bot instance can be passed to a move function to test its return value.
-    The layout is a string that can be passed to create_layout."""
-    if game is not None:
-        raise RuntimeError("Re-using an old game is not implemented yet.")
+    """Setup a test game environment useful for testing move functions.
+
+    Parameters
+    ----------
+    layout : str
+          a valid layout string, like the one obtained by print(bot). For example:
+              layout='''
+                     ########
+                     #0    .#
+                     #.1  EE#
+                     ########
+                     '''
+
+    is_blue : bool
+           when True setups up the game assuming your bots are in the blue team,
+           i.e. the team with the homezone on the left side of the maze. Otherwise
+           your bots are assumed to be in the red team.
```

Comment:
I think ‘Otherwise’ is misleading here as it implies that this also happens when you leave it out. But the default is True. So, I’d just write ‘when False’.